lrrbot/commands/static.py

```diff
@@ -1,86 +1,143 @@
+		if row is None:
+			return None
+		else:
+			return row[0]
+
+def get_response(bot, command):
+	responses = bot.metadata.tables["commands_responses"]
+
+	with bot.engine.connect() as conn:
+		command_id = _get_command_id(bot, conn, command)
+		if command_id is None:
+			return None
+
+		query = sqlalchemy.select(responses.c.response).where(responses.c.command_id == command_id)
+		row = common.utils.pick_random_elements(conn.execute(query), 1)[0]
+
+		if row is None:
+			return None
+		else:
+			return row[0]
+
+def get_advice(bot):
+	return get_response(bot, ADVICE)
+
+def generate_docstring(bot):
+	commands = bot.metadata.tables["commands"]
+	aliases = bot.metadata.tables["commands_aliases"]
+	responses = bot.metadata.tables["commands_responses"]
+
 	def generator():
-		for (response, access), command in inverse_responses.items():
-			fragment = ""
-			if isinstance(command, list):
-				for cmd in sorted(command):
-					fragment += "Command: %s%s\n" % (config["commandprefix"], cmd)
-			else:
-				fragment += "Command: %s%s\n" % (config["commandprefix"], cmd)
-			fragment += "Throttled: 30\n"
-			fragment += "Throttle-Count: 2\n"
-			fragment += "Literal-Response: true\n"
-			if access == "sub":
-				fragment += "Sub-Only: true\n"
-			elif access == "mod":
-				fragment += "Mod-Only: true\n"
-			fragment += "Section: text\n"
-			fragment += "\n"
-			response = response if isinstance(response, str) else response[0]
-			fragment += response + "\n"
-			yield fragment
-	return "\n--command\n".join(generator())
-
-def generate_expression():
-	return "(%s)" % "|".join(re.escape(c).replace("\\ ", " ") for c in storage.data["responses"])
+		with bot.engine.connect() as conn:
+			query = sqlalchemy.select(commands.c.id, commands.c.access, aliases.c.alias)
+			query = query.select_from(commands.join(aliases, aliases.c.command_id == commands.c.id))
+			query = query.order_by(commands.c.id, aliases.c.id)
+
+			first = True
+			for command_id, rows in itertools.groupby(conn.execute(query), key=lambda row:row[0]):
+				if not first:
+					yield "--command"
+				first = False
+
+				for _, access, alias in rows:
+					yield f"Command: {config['commandprefix']}{alias}"
+				yield "Throttled: 30"
+				yield "Throttle-Count: 2"
+				yield "Literal-Response: true"
+				if access == ACCESS_SUB:
+					yield "Sub-Only: true"
+				elif access == ACCESS_MOD:
+					yield "Mod-Only: true"
+				yield "Section: text"
+				yield ""
+
+				resp_query = sqlalchemy.select(responses.c.response).where(responses.c.command_id == command_id)
+				resp_query = resp_query.order_by(responses.c.id).limit(1)
+				response, = conn.execute(resp_query).first()
+				yield response
+
+	return "\n".join(generator())
+
+def generate_expression(bot):
+	aliases = bot.metadata.tables["commands_aliases"]
+	with bot.engine.connect() as conn:
+		query = sqlalchemy.select(aliases.c.alias)
+		aliases = conn.execute(query)
+		aliases = [alias for alias, in aliases]
```

Comment:
This is [`Result.scalars()`](https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Result.scalars).
```suggestion
		aliases = conn.execute(query).scalars().all()
```